validator/tests/test_merkle_trie/tests.py

```diff
@@ -140,6 +140,89 @@ def test_merkle_trie_update(self):
             with self.assertRaises(KeyError):
                 self.get(address, ishash=True)
 
+    # This UT is similar to update except that it will
+    # ensure that there are no index errors in path_map
+    # within update function in case there are addresses
+    # within set_items & delete_items which have a common
+    # prefix(of any length).
+
+    # A merkle tree is cretaed with some intial values
+    # which is then updated(set & delete).
```

Comment:
Great explanatory comment. Can you please move it under the method declaration with the triple quote docstring style like this: https://github.com/hyperledger/sawtooth-core/blob/master/validator/tests/test_config/tests.py#L28-L48